activity_calendar/templates/activity_calendar/slot_blocks/slot_block.html

```diff
@@ -0,0 +1,45 @@
+{% load static %}
+{% load calendar_tags %}
+
+<div class="card greenify activity-wide">
+    <div class="row no-gutters">
+        <div class="col-2 col-sm-1 activityImage">
+            <div class="heightener"></div>
+            <img src="{{slot.image_url}}" alt="" class="card-img">
+        </div>
+        <div class="col-10 col-sm-11 col-md-9">
+            <div class="card-body activityName">
+                <h5 class="card-title">
+                    {{ slot.title }}
+                </h5>
+                {% if slot.location %}
+                    <div class="row no-gutters">
+                        <small class="text-muted">
+                            Alternative Location: {{ slot.location }}
+                        </small>
+                    </div>
+                {% endif %}
+                <p class="card-text">{{ slot.description }}</p>
+
+                <div class="row no-gutters">
+                    <div class="card-text">
+                        <small class="text-muted">Participants:
+                            {% if slot.max_participants != -1 %}
+                                {{ slot.participants.all|length }} / {{ slot.max_participants }}
+                            {% else %}
+                                {{ slot.participants.all|length }} / &infin;
```

Comment:
Why show the infinity symbol here instead of omitting the limit completely if there is no limit?

So you'd have `Participants: 7` instead of `Participants: 7 / &infin;`

Reply:
Style continuity. Figured this might be a bit clearer that there is no cap without saying infinite particiapnts (which it said earlier)